images/win/scripts/Installers/Install-RootCA.ps1

```diff
@@ -104,6 +104,27 @@ function Disable-RootAutoUpdate {
     Set-ItemProperty $regPath -Name $regKey -Type DWord -Value 1
 }
 
+function Invoke-WithRetry {
+     param([ScriptBlock]$Command, [ScriptBlock] $BreakCondition, [int] $RetryCount=4, [int] $Sleep=1)
+
+     $c = 0
+     while($c -lt $RetryCount -or $c -eq 0){
```

Comment:
nitpick: remove `-or $c -eq 0` condition

Reply:
done. this remove the case with RetryCount=0, in that case the command is never executed.